BootloaderCommonPkg/Include/Library/ShellLib.h

```diff
@@ -10,6 +10,9 @@
 
 #include <PiPei.h>
 #include <Library/BaseLib.h>
+#include <Library/ConsoleOutLib.h>
+
+#define ShellPrint(Format, ...)     ConsoleDebugPrintUnicode (Format, ##__VA_ARGS__)

```

Comment:
can be simpler  as "#define  ShellPrint   ConsoleDebugPritnUnicode" ,   :)

Reply:
Yes right :) Thanks!